osidb/models.py

```diff
@@ -2919,6 +2933,45 @@ def validate(self, *args, **kwargs):
         super().clean_fields(*args, **kwargs)
 
 
+class Package(models.Model):
+    """Model representing a package with connected versions."""
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    flaw = models.ForeignKey(
+        Flaw,
+        related_name="package_versions",
+        on_delete=models.CASCADE,
+    )
+
+    package = models.CharField(max_length=2058)
+
+    # TODO: This is temporarily present to make existing tests happy. Reconsider whether to delete.
+    def validate(self, *args, **kwargs):
+        super().clean_fields(*args, **kwargs)
```

Comment:
Yes, it only says this. Since it is refactored away in follow-up commits, I don't see value in writing a fully-accurate explanation.

The short is that old models had `.validate()` and old tests ran the validation.

I made gradual commits/changes so as to make sure I didn't introduce a bug into both new models and new tests at the same time. So this is from a state where old tests (with changes as small as possible) ran against the new models.

Reply:
Also it seems this is from a state where the custom osidb mixins were not introduced to the new models yet. They have their own `validate()` IIRC.